File: poker-after-dark-specification/src/main/java/com/slusarz/pokerafterdark/specification/model/game/Game.java
package com.slusarz.pokerafterdark.specification.model.game;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDate;
import java.util.List;

@Getter
@Setter
@Builder
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Game {

    private String id;

    private Host host;

    private LocalDate date;

    private int pot;

    private int participantsCount;

    private List<Participant> participants;

}

File: poker-after-dark-web/src/app/shared/models/game/game.ts
import { Host } from "./host"
import { Participant } from "./participant";
import { GameActions } from "./game-actions";

export class Game {

    id: string;

    actions: GameActions;

    type: string;

    host: Host;

    date: string;

    pot: number;

    participantsCount: number;

    participants: Participant[];

}
